src/main/kotlin/org/rust/lang/core/psi/ext/RsElement.kt

```diff
@@ -131,17 +131,9 @@ abstract class RsStubbedElementImpl<StubT : StubElement<*>> : StubBasedPsiElemen
 }
 
 fun RsElement.findInScope(name: String, ns: Set<Namespace>): PsiElement? {
-    var resolved: PsiElement? = null
-    val processor = createProcessor(name) { entry ->
-        if (entry.name == name) {
-            resolved = entry.element
-            true
-        } else {
-            false
-        }
+    return pickFirstResolveVariant(name) {
```

Comment:
It seems that it's ok! I've checked each usage of `findInScope` and it seems that checking `cfg` fixes some bug in every case! But existence of such API is controversial anyway. I think this function should return `List<PsiElement>` because of possible multiresolve and under the hood delegate to normal path name resolution instead of such ad-hoc solutions.